packages/common/src/types/filterGrammar.test.ts

```diff
@@ -90,6 +90,20 @@ describe('attachTypesToModels', () => {
         expect(parser.parse(' >=32')).toEqual(expected);
         expect(parser.parse(' >= 32')).toEqual(expected);
     });
+
+    it('Is null', async () => {
```

Comment:
can you escape the null value to match the exact string "null"? I don't think it's a requirement but what's the current behaviour?

Reply:
Not sure what you mean, but this parser method takes only a  string as an argument, to process `null`s I don't use `peg` https://github.com/lightdash/lightdash/pull/5729/files#diff-546a539fae0c7fc3cf99abec54c5cb175292ec43d5f4525a15bdb84080ba0be5R218

Reply:
I mean would be possible to create a filter that's like `myvalue === "null"` rather than `myvalue === null`